desktop_native/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "desktop_native"
+version = "0.1.0"
+license = "GPL-3.0"
+edition = "2018"
+exclude = ["index.node"]
+
+[lib]
+crate-type = ["cdylib"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tokio = { version = "1.17.0", features = ["full"] }
+
+[dependencies.neon]
+version = "0.10.0-alpha.3"
+default-features = false
+features = ["napi-6", "promise-api", "task-api"]
+
+[target.'cfg(windows)'.dependencies.windows]
+version = "0.32.0"
+features = [
+    "alloc",
+    "Foundation",
+    "Security_Credentials_UI",
+    "Win32_Foundation",
+    "Win32_System_WinRT",
+]
```

Comment:
You can't have newlines in toml, so that would be a fairly long line.
```toml
neon = { version = "0.10.0-alpha.3", default-features = false, features = ["napi-6", "promise-api", "task-api"] }
```

Only reason to have it in a separate table.

Reply:
You can, you just need to split it where it accepts them

```toml
neon = { version = "0.10.0-alpha.3", default-features = false,features = [
  "napi-6",
  "promise-api",
  "task-api",
] }
```

although I don't really like that syntax